app/common/FirebaseService.ts

```
@@ -25,107 +24,168 @@
-      })
-    );
+    return promiseArr;
   }
 
-  private async fetchNibbles(): Promise<void> {
-    const q: Query<DocumentData, DocumentData> = query(
-      collection(this.firestore, nibblesCollection.name),
-      where(nibblesCollection.isPublishedField, "==", true),
-      orderBy(nibblesCollection.publishDateField, "desc")
-    );
+  private async fetchNibbles(): Promise<void[]> {
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)